v3/cypress/e2e/axis.spec.ts

```diff
@@ -15,18 +15,16 @@ const arrayOfValues = [
   { attribute: "Mass", values: [ "0", "1000", "2000", "3000", "4000", "5000", "6000", "7000" ]},  // Y
   { attribute: "Sleep", values: [...Array(12).keys()].map(i => `${2 * i}`)},  // Y
   { attribute: "Speed", values: [ ]},
-  { attribute: "Habitat", values: [ "both", "land", "water" ]},
-  { attribute: "Diet", values: [ "both", "meat", "plants" ]},
+  { attribute: "Habitat", values: [ "land", "water", "both" ]},
+  { attribute: "Diet", values: [ "plants", "meat", "both" ]},
```

Comment:
We apparently didn't notice these obvious errors because the code for testing these values wasn't running. See comment in `axis-helper.ts` for details.